File: frontend_mycoffeedau/src/models/voucher.ts
export interface Voucher {
  id: string;
  content: string;         // Nội dung / mô tả voucher (VD: "Giảm 20% đơn hàng")
  pointRequirement: string; // Yêu cầu hạng thành viên (VD: "Tập Sự", "Vàng")
  point: number;           // Số điểm cần để đổi
}

File: backend_mycoffeedau/backend_mycoffeedau/Model/Voucher.cs
namespace backend_mycoffeedau.Model
{
    public class Voucher
    {
        public string Id { get; set; } = string.Empty; // Khớp NVARCHAR(50)
        public string Content { get; set; } = string.Empty;
        public string PointRequirement { get; set; } = string.Empty;
        public int Point { get; set; } = 0;
    }
}
